### PR TITLE
Feature/add security to standards

### PR DESCRIPTION
**Pull request by duvaneljulien**

### CODE REVIEW

File: en/security.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  3. _Data Access_: Does the API allow data creation, modification or deletion ?
+
+These questions and their answers *MUST* be included in the documentation for each API (see General Documentation chapter). In addition, the column “Security Case” (internet, sensitive data, data update) in the document headers *MUST* be filled.
+
+If an API has security criteria, then the following requirements *MUST* be met.
+ 
+  - Integration into the API manager (mandatory if the API is exposed to the Internet, recommended otherwise)   
+  - Rights managed up to the method level and not just at the API level, And the access model to be documented  (IAM sheet) (covered by the API Manager)   
+  - Authenticated API calls using tokens (covered by API Manager)   
+  - Logging – API calls must be logged and kept for 90 days (covered by the API Manager, with checks for log management and service logs)
```
Comment (hugo-ma-alves) [inline en/security.md:17]: Not sure if I understood it correctly. But do the services also have to keep the application /access logs for 90Days? Or is it only the API manager logs?
 I Believe we don't have a long retention period on our side.

**Review by jeperon-vaudoise — APPROVED** [review]

File: en/security.md (hunk 1 above)
Comment (duvaneljulien) [reply en/security.md:17]: Good question, I'll ask our security colleagues to answer.

**Review by jfreyre — APPROVED** [review]

**Review by sebrouss — APPROVED** [review]

File: en/security.md (hunk 1 above)
Comment (duvaneljulien) [reply en/security.md:17]: @hugo-ma-alves : It's in french (sorry :-/), but here is the answer.

If there are logs at the API Manager level, then it's perfect for our security colleagues. 

image

File: fr/sécurité.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Ces questions et leurs réponses *DOIVENT* figurer dans la documentation de chaque API (voir chapitre Documentation Générale). De plus, la colonne "Security Case" (internet, sensitive data, data update) dans les en-têtes de la documentation *DOIT* être renseignée.
+
+Si une API présente des enjeux de sécurité, alors les exigences suivantes *DOIVENT* être remplies.
+ 
+  - Code scanning – intégration du code source à WhiteHat (seulement si l'API est exposée sur Internet - les _repositories_ git liés à l'API doivent être documentés et fournis à l'équipe sécurité)
+  - Intégration dans l'API manager (obligatoire si API exposée sur Internet, recommandé sinon)
+  - Droits gérés au niveau des méthodes et pas seulement de l'API et modèle d'accès documenté (fiche IAM) (couvert par l'API Manager)
+  - Appel à l'API authentifié via jeton (couvert par l'API Manager)
+  - Logging - les appels à l'API doivent être enregistrés et conservés pendant 90 jours (couvert par l'API Manager, mais vérifier gestion des logs et vérifier avec les logs du service)
+  - Mise en place du contrôle périodique des droits (CPA) pour une revue régulière des accès donnés sur l’API aux end users dans l'API Manager
```
Comment (yrandin) [inline fr/sécurité.md:19]: Détail: Utiliser un ' au lieu du ’

**Review by yrandin — APPROVED** [review]

**Review by jfreyre — APPROVED** [review]

**Review by tdespont — APPROVED** [review]

File: en/security.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+An API is considered to have security criteria if the answer to at least one of the following 3 questions is “yes”:
+  1. _Visibility_: Is the API exposed to outside of Vaudoise ?
+  2. _Data Type_: Does the API allow access to restricted or confidential data according to Vaudoise confidentiality definition levels ?
+  3. _Data Access_: Does the API allow data creation, modification or deletion ?
+
+These questions and their answers *MUST* be included in the documentation of each API (see General Documentation chapter). Documentation *MUST* also indicate if the API _present security issues_.
+
+If an API has security criteria, then the following requirements *MUST* be met.
+ 
+  - Integration into the API manager (mandatory if the API is exposed to the Internet, recommended otherwise)
```
Comment (mimi06500) [inline en/security.md:14]: If I'm not mistaken, the part of the code scanning is missing here

**Review by mimi06500 — APPROVED** [review]

**Review by SebLbu — APPROVED** [review]

File: en/security.md (hunk 3 above)
Comment (duvaneljulien) [reply en/security.md:14]: Wow, nice one. Thank you very much.

I'll update the file.

**Pull request merged**